packages/react-components/theme-designer/src/components/Palette/Palette.tsx

```diff
@@ -15,10 +17,30 @@ const useStyles = makeStyles({
     justifyContent: 'space-evenly',
   },
   block: {
+    display: 'grid',
+    gridTemplateColumns: '0.5em auto',
+    gridTemplateRows: '0.5em 1fr 1fr 0.5em',
+    flexGrow: 1,
+    flexShrink: 0,
+    ':hover': {
+      flexShrink: 1,
+    },
+    ':hover > div': {
+      justifyContent: 'space-between',
```

Comment:
small nit, but the hover style should just change the display, the other styles can stay on the default style